plugins/deploy_waitlist/app/controllers/deploy_waitlist_controller.rb

```diff
@@ -0,0 +1,42 @@
+class DeployWaitlistController < ApplicationController
+
+  def show
+    respond_to do |format|
+      format.html
+      format.json { render json: current_waitlist.to_json }
+    end
+  end
+
+  def add
```

Comment:
do we want to consider RESTful routes? I think add and remove make sense, but i'm just asking if we have strong opinions either way